src/device.coffee

```diff
@@ -186,7 +186,8 @@ exports.getOSVersion = ->
 		for line in lines
 			[ key, val ] = line.split('=')
 			releaseItems[_.trim(key)] = _.trim(val)
-		return releaseItems['PRETTY_NAME']
+		# Remove enclosing quotes: http://stackoverflow.com/a/19156197/2549019
+		return releaseItems['PRETTY_NAME'].replace(/^"(.+(?="$))"$/, '$1')
```

Comment:
The first would fail if the string was something like `Ubuntu 18.04 "cool cactus"`, the second one works but the stackoverflow answer mentioned reports this one is slightly faster.